pkg/runner/local_docker.go

```diff
@@ -548,6 +551,11 @@ func (r *LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow *rp
 				},
 			}
 
+			ulimits, err := conv.ToUlimits(cfg.Ulimits)
```

Comment:
This applies the configured ulimits to plan contaienrs, but not for "infrastructure" containers, redis, etc. because those containers are created during healthcheck fix actions and the input configuration isn't passed there. I suspect we will want that to be configured by the same option, which will require passing the RunInput composition.

Reply:
Infrastructure containers are not run-scoped; they have their own configuration.